solutions/59.js

```diff
@@ -0,0 +1,15 @@
+//Function that takes in a number, returns true if number is prime. False otherwise.
+const solution = (x) => {
+  if(x<2){
+    return false;
+  }
+  for(i=x; i>=1; i--){
```

Comment:
`for( let i = x ...)` Bad practice to assign value to i without declaring it first